sdk-base/src/main/java/com/mapbox/maps/plugin/viewport/experimental/state/OverviewViewportState.kt

```diff
@@ -0,0 +1,13 @@
+package com.mapbox.maps.plugin.viewport.experimental.state
+
+import com.mapbox.maps.plugin.viewport.experimental.data.OverviewViewportStateOptions
+
+/**
+ * The [ViewportState] that shows the overview of a given geometry.
+ */
+interface OverviewViewportState : ViewportState {
+  /**
+   * Describes the configuration options of the state.
+   */
+  var options: OverviewViewportStateOptions
+}
```

Comment:
Interesting. I could do this on iOS too. So far I haven't defined a protocol for this. Are we comfortable with restricting any additions be contained within options at least until the next major version? I don't have any problem with it, but I wanted to ask anyway.

Reply:
yes, I think we are good with the flexibility to make additions to the options for now.